appinventor/components/src/com/google/appinventor/components/runtime/CloudDB.java

```diff
@@ -808,6 +782,59 @@ public void run() {
     }
   }
 
+  /**
+   * Gets the specified value from the underlying Redis database, or
+   * returns the specified value if the tag is not present.
+   *
+   * <p>The value is returned as an AtomicReference, and will contain
+   * a null value in case of exceptions.
+   *
+   * @param tag  tag of the value to get
+   * @param valueIfTagNotThere  value to set to the reference if tag is not present
+   * @return  AtomicReference containing the indicated value
+   */
+  private AtomicReference<Object> getValueByTag(final String tag, final Object valueIfTagNotThere) {
+    AtomicReference<Object> value = new AtomicReference<Object>();
+
+    Jedis jedis = getJedis();
+    try {
+      if (DEBUG) {
+        Log.d(LOG_TAG,"about to call jedis.get()");
+      }
+      String returnValue = jedis.get(projectID + ":" + tag);
+      if (DEBUG) {
+        Log.d(LOG_TAG, "finished call jedis.get()");
+      }
+      if (returnValue != null) {
+        String val = JsonUtil.getJsonRepresentationIfValueFileName(returnValue);
```

Comment:
This call is obsolete. call JsonUtil.getJsonRepresentationIfValueFileName(form, returnValue). I.e., add form as the first argument.

Reply:
@jisqyv Fixed.